### ISSUE TITLE
Admin-panel allows the creation of event-reservation with null allergies, even though the model doesn't allow it

### ISSUE THREAD

**Issue by henrikhorluck** [dotkom/onlineweb4#2274]
https://sentry.io/organizations/dotkom/issues/1052986616/?referrer=github_plugin

```
NotNullViolation: null value in column "allergies" violates not-null constraint
DETAIL:  Failing row contains (279, 28, Alis WW, reserverte plasser til nutrix, null).

  File "django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)

IntegrityError: null value in column "allergies" violates not-null constraint
DETAIL:  Failing row contains (279, 28, Alis WW, reserverte plasser til nutrix, null).

(27 additional frame(s) were not displayed)
...
  File "reversion/admin.py", line 159, in add_view
    return super(VersionAdmin, self).add_view(request, form_url, extra_context)
  File "apps/events/models.py", line 1049, in save
    super().save(force_insert=force_insert, force_update=force_update, using=using, update_fields=update_fields)

IntegrityError: null value in column "allergies" violates not-null constraint
DETAIL:  Failing row contains (279, 28, Alis WW, reserverte plasser til nutrix, null).

```

**Comment by henrikhorluck** [comment]
Because of how the reservations work, it might make sense to not have it required

**Comment by sklirg** [comment]
`File "reversion/admin.py", line 159, in add_view`

This is the reversion app, not django admin.

If looking at the history of an element it might raise an exception if you try to view the instance of it which doesn't pass validation or doesn't have the fields required, as well as trying to save such a version.

Since the models continue to change over time, it's hard to keep a consistent reversion history of them.

I don't think there's an easy way to fix this.

**Comment by oleast** [comment]
fixed by #2313
